src/main/java/org/opentripplanner/middleware/controllers/api/MonitoredTripController.java

```diff
@@ -27,11 +35,54 @@ public MonitoredTripController(String apiPrefix) {
     MonitoredTrip preCreateHook(MonitoredTrip monitoredTrip, Request req) {
         verifyBelowMaxNumTrips(monitoredTrip.userId, req);
         monitoredTrip.initializeFromItinerary();
+        ItineraryExistenceChecker.Result checkResult = checkItineraryExistence(monitoredTrip, req);
```

Comment:
Is it worth creating E2E tests for this, similar to ApiUserFlowTest?

Reply:
Cast your votes for adding e2e tests.